tests/dump_test.go

```diff
@@ -294,63 +291,10 @@ func runTestWithCfg(t *testing.T, name string, cfg *litter.Options, cases ...int
 			return
 		}
 
-		assertEqualStringsWithDiff(t, string(reference), dump)
+		assert.Equal(t, string(reference), dump)
 	})
 }
 
-func runTests(t *testing.T, name string, cases ...interface{}) {
+func runTests(t *testing.T, name string, cases ...any) {
 	runTestWithCfg(t, name, &standardCfg, cases...)
 }
-
-func diffStrings(t *testing.T, expected, actual string) (*string, bool) {
```

Comment:
To be clear, it was very intentional to use `diff` here. `assert.Equal()` produces super unreadable unified diffs. If you want different output, I'd be fine with an environment variable to control it (defaulting to `diff`).

Reply:
Reverted, fwiw the pure go diff is a _lot_ faster